mods/d2k/weapons/largeguns.yaml

```diff
@@ -89,7 +89,7 @@ DevBullet:
 		Blockable: false
 		Shadow: true
 		LaunchAngle: 62
-		Inaccuracy: 768
+		Inaccuracy: 256 # The siege tank of the original somehow doesn't seem to be any more inaccurate than a normal combat tank, but following the ORA tradition we increase the inaccuracy because this is an artillery unit.
```

Comment:
Can we remove this comment, please?

Reply:
We can, but if I remove it I want to first have a final decision that we're going to have increased inaccuracy on the Siege tank compared to the original.

Reply:
we should leave it to a future balance pass, IMO.

Reply:
OK, will set it to match the original then.

Reply:
Yeah, I'd stick with the original value for now (same then goes for the Spread, in case you increased that to make up for the higher inaccuracy).

Reply:
Removed the override along with the comment.